vividus-analytics/src/main/resources/spring.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:util="http://www.springframework.org/schema/util"
+    xsi:schemaLocation="
+        http://www.springframework.org/schema/beans https://www.springframework.org/schema/beans/spring-beans.xsd
+        http://www.springframework.org/schema/util https://www.springframework.org/schema/util/spring-util.xsd"
+    default-lazy-init="true">
+
+    <bean id="analyticsFacade" init-method="init" class="org.vividus.analytics.GoogleAnalyticsFacade">
+        <property name="trackingId" value="${analytics.tracking-id}" />
```

Comment:
afaik we are moving in direction of replacing setter injection with constructor injection

Reply:
but not for properties